PARQUET-2369: Add new test file concatenated_gzip_members.parquet (#41)
apache/parquet-testing: data/README.md

| plain-dict-uncompressed-checksum.parquet         | uncompressed and dictionary-encoded INT32 and STRING columns in format v1 with a matching CRC |
| rle-dict-uncompressed-corrupt-checksum.parquet   | uncompressed and dictionary-encoded INT32 and STRING columns in format v2 with a mismatching CRC |
| large_string_map.brotli.parquet       | MAP(STRING, INT32) with a string column chunk of more than 2GB. See [note](#large-string-map) below |

> | concatenated_gzip_members.parquet     | 513 int64 numbers compressed using 2 gzip members |

Reviewer: ```suggestion
| concatenated_gzip_members.parquet     | 513 UINT64 numbers compressed using 2 gzip members |
```

Author: why you choosed [U]INT64 ?
parquet-mr shows INT64:

          type      encodings count     avg size   nulls   min / max
long_col  INT64     G RB_     513       2.86 B             "1" / "513"

Reviewer: The schema shows that its logical type is unsigned.
```
message root {
  optional int64 long_col (INTEGER(64,false));
}
```

Author: yes, I missed the logical type. thanks

| large_string_map.brotli.parquet       | MAP(STRING, INT32) with a string column chunk of more than 2GB. See [note](#large-string-map) below |
| float16_nonzeros_and_nans.parquet | Float16 (logical type) column with NaNs and nonzero finite min/max values |
| float16_zeros_and_nans.parquet    | Float16 (logical type) column with NaNs and zeros as min/max values. . See [note](#float16-files) below |

> | concatenated_gzip_members.parquet     | 513 UINT64 numbers compressed using 2 concatenated gzip members (A GZIP member refers to a single compressed file or stream that has been compressed using the GZIP algorithm.) |

Reviewer 2: ```suggestion
| concatenated_gzip_members.parquet     | 513 UINT64 numbers compressed using 2 concatenated gzip members in a single data page |
```
